.github/workflows/ci.yml

```diff
@@ -51,6 +51,11 @@ jobs:
             calculate-code-coverage: 'yes'
             phpunit-flags: '--testsuite coverage --exclude-group covers-nothing --coverage-clover build/logs/clover.xml'
 
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.1'
+            composer-flags: '--ignore-platform-req=php' # as this is a version not yet officially supported by PHP CS Fixer
```

Comment:
is it needed? we no longer check upper version via composer.

Reply:
we don't, but others do:
```
/(...)/PHP-CS-Fixer(2.19)$ composer why-not php ^8.1
phpspec/prophecy  1.13.0  requires  php (^7.2 || ~8.0, <8.1)
```